### ISSUE TITLE
Bump asdf version

### ISSUE THREAD

**Issue by andys8** [doughsay/omf-asdf#1]
### Issue

`asdf` version is hardcoded to `v0.5.1`. Currently `v0.7.1` is the newest stable version: https://github.com/asdf-vm/asdf/releases

https://github.com/doughsay/omf-asdf/blob/1da927fb3ce6b8161a8eed6600a71795f3dc44fa/hooks/install.fish#L6

### Solution

I manually updated `asdf` and its working fine for me.

The version can either be bumped or the install script changed to install the checkout version:

```shell
git clone https://github.com/asdf-vm/asdf.git ~/.asdf
cd ~/.asdf
git checkout "$(git describe --abbrev=0 --tags)"
```
See https://asdf-vm.com/#/core-manage-asdf-vm

**Comment by doughsay** [comment]
Thanks, I'll look into using the proposed solution.

**Comment by doughsay** [comment]
Fixed in #2, thanks @andys8!
